src/psyclone/psyGen.py

```diff
@@ -5140,6 +5148,34 @@ def scope(self, new_scope):
 
         self._scope = new_scope
 
+    def gen_c_definition(self):
+        '''
+        Generates string representing the C language definition of the symbol.
+
+        :return: The C definition of the symbol.
+        :rtype: str
+        '''
+        code = ""
+        if self.datatype == "real":
+            code = code + "double "
+        elif self.datatype == "integer":
+            code = code + "int "
+        elif self.datatype == "character":
+            code = code + "char "
+        else:
+            raise NotImplementedError(
+                "Could not generate the C definition for the variable '{0}', "
+                "type '{1}' is currently not supported."
+                "".format(self.name, self.datatype))
+
+        # If the argument is an array, in C language we define it
+        # as an unaliased pointer.
+        if len(self.shape) > 0:
```

Comment:
pylint complains about this (should be `if self.shape:`)

Reply:
Fixed